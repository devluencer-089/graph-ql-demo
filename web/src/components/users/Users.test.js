import React from 'react';
import { createShallow } from 'material-ui/test-utils';
import Users from './Users';

describe('Users Component', () => {
  let shallow = createShallow();

  it('should match snapshot', () => {
    let eut = shallow(<Users />);

    expect(eut).toMatchSnapshot();
  });
});
