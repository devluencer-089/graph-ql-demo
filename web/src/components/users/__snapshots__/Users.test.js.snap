// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users Component should match snapshot 1`] = `
<div>
  Users
</div>
`;
